OpenTrans.net/ProductPriceFix.cs

```diff
@@ -16,15 +16,13 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-using System;
-using System.Collections.Generic;
-using System.Text;
 
 namespace OpenTrans.net
 {
     public class ProductPriceFix
     {
-        public decimal PriceAmount {get;set;}
+        public decimal PriceAmount { get; set; }
         public decimal PriceQuantity { get; set; }
+        public PriceBaseFix PriceBaseFix { get; set; }
     }
```

Comment:
could you add the remaining elements here as well?
According to the specification:

![Uploading grafik.png…]()